src/components/CarouselSection/CarouselCard/CarouselCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import stl from "./CarouselCardStyle.module.scss"
+import { starIcon} from "../../DummyData/CarouselSection";
+import Image from 'next/image'
+
+type CarouselCardProps = {
+  description: string;
+  clientImg: string;
+  clientName: string;
+  clientProfile: string;
+};
+
+const CarouselCard = ({description, clientImg, clientName, clientProfile}:CarouselCardProps) => {
+  return (
+    <>
```

Comment:
Empty tags not required for single child.

Reply:
Removed